core/hydromet_stratified.py

```diff
@@ -383,6 +381,169 @@ def Scenarios_low_and_high_S(df_runoff_SR1: pd.DataFrame, mu: float, GEV_paramet
                                    'Avg. S (Upper 50%)', 'Rainfall (Upper 50%)'])
     return df_SR2
 
+def precip_hyetograph_nrcs(df : pd.DataFrame) -> pd.DataFrame:
```

Comment:
I compared this function to the equations in the Nested_Rainfall_Distribution_Spreadsheet_template.xlsx since I am not personally familiar with this method though I get the idea.